toDoApp/src/screens/home/ClearAllDone/index.tsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react'
 import { styles } from './styles'
-import { Pressable, Text, Alert } from 'react-native'
+import { Pressable, Text } from 'react-native'
+import { constants } from './strings'
```

Comment:
i see in other places and its more like a suggestion but what do you think to improve a bit the names.
instead of `constants` what if we called this for what they are `strings` , do you feel it more suitable?